public/scripts/models/app.js

```diff
@@ -11,21 +11,18 @@
     this.link = argument.link;
     this.img = argument.img;
   }
-  //projects page//
+  //projects page initialization, appending each project to the dom //
```

Comment:
I would normally dock points for this, but for the scope of this course, great job adding comments for yourself so you know what's happening in your code. 👍